awx/main/management/commands/dump_auth_config.py

```diff
@@ -40,6 +41,15 @@ class Command(BaseCommand):
         "USER_SEARCH": False,
     }
 
+    def is_enabled(self, settings, keys):
```

Comment:
I think for clarity it might be best to return 2 values, the bool and the missing fields

if no missing fields, `return True, None`
if missing fields, `return False, missing_fields`